src/components/grid/GridComponent.vue

```diff
@@ -54,9 +54,7 @@
 
           >
             <th>
-              <span class="variable-title">
-                {{variableName(gridVariables[rowIndex])}}
-              </span>
+              <grid-titel-info :info="gridVariables[rowIndex]" />
```

Comment:
Suggest to use slot instead of property this would allow changing or styling the variable stuff a the grid level

Reply:
I don't see that working while holding the responsibility of choosing name or label  within this component. See line 25. Suggestions?